pyflow/RCDT2/Nodes/node_loader.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from RCDT2.Nodes.core import Service
+
+
+def get_pyflow_nodes_from_ros_services(services: list[tuple[str, str]]) -> dict:
+    pyflow_nodes = {}
+    for service in services:
+        service_name = service[0]
+        service_type = service[1]
+        pyflow_nodes[service_name] = create_class_from_service(service_type)
+    return pyflow_nodes
```

Comment:
Deze hele functie kan zelfs vrij makkelijk in een behoorlijk leesbare oneliner:

```suggestion
    return {service_name: create_class_from_service(service_type) for service_name, service_type in services}
```